static/app/components/replays/player/useTimelineMouseTracking.tsx

```diff
@@ -11,43 +11,18 @@ type Opts<T extends Element> = {
   elem: RefObject<T | null>;
 };
 
-export function useScrubberMouseTracking<T extends Element>({elem}: Opts<T>) {
-  const replay = useReplayReader();
-  const [, setCurrentHoverTime] = useCurrentHoverTime();
-  const durationMs = replay?.getDurationMs();
-
-  const handlePositionChange = useCallback(
-    (params: any) => {
-      if (!params || durationMs === undefined) {
-        setCurrentHoverTime(undefined);
-        return;
-      }
-      const {left, width} = params;
-
-      if (left >= 0) {
-        const percent = left / width;
-        const time = percent * durationMs;
-        setCurrentHoverTime(time);
-      } else {
-        setCurrentHoverTime(undefined);
-      }
-    },
-    [durationMs, setCurrentHoverTime]
-  );
```

Comment:
there were two versions of this before.. the removed version isn't aware of scaling. But if we hard-code the scale to `1` then they work the same.